core/trino-main/src/main/java/io/trino/operator/window/pattern/MeasureComputation.java

```diff
@@ -41,7 +40,16 @@
     private final PageProjection projection;
 
     // value accessors ordered as expected by the compiled projection
-    private final List<PhysicalValuePointer> expectedLayout;
+    private final List<PhysicalValueAccessor> expectedLayout;
+
+    private final MatchAggregation[] aggregations;
+
+    // precomputed `Block`s with null values for every `PhysicalValuePointer`.
```

Comment:
Backticks don't mean anything in java comments -- it's not markdown, so they don't get formatted. They just make the text a little harder to read, so remove them.